samples/drivers/audio/speaker/src/main.c

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright (c) , ithinx GmbH
+ *
+ * SPDX-License-Identifier: Proprietary
```

Comment:
this will need to be updated as well, but it looks like you have this as a TODO already :)

Reply:
done